### PR TITLE
Update yaml to include paramtools

### PR DESCRIPTION
**Pull request by jdebacker**

### CODE REVIEW

File: conda.recipe/meta.yaml [hunk 1]
```diff
@@ -8,14 +8,16 @@ source:
...
+    - paramtools
     - xlrd
 
   run:
     - python
-    - "taxcalc>=2.1.0"
+    - "taxcalc>=2.4.0"
     - setuptools
+    - paramtools
     - xlrd
```
Comment (hdoupe) [inline conda.recipe/meta.yaml]: @jdebacker you should specify "paramtools>=0.7.1" right here.

Comment (jdebacker) [reply conda.recipe/meta.yaml]: Got it - thanks.

File: conda.recipe/meta.yaml [hunk 2]
```diff
@@ -8,14 +8,17 @@ source:
 requirements:
   build:
     - python
-    - "taxcalc>=2.1.0"
+    - "taxcalc>=2.4.0"
+    - "paramtools>=0.7.1"
     - setuptools
+    - paramtools
```
Comment (hdoupe) [inline conda.recipe/meta.yaml]: An extra "paramtools" is in the build list.

Comment (jdebacker) [reply conda.recipe/meta.yaml]: Doh!

Comment (hdoupe) [reply conda.recipe/meta.yaml]: Ha! LGTM

**Pull request merged**